code/modules/power/supermatter/supermatter.dm

```diff
@@ -90,6 +90,22 @@
 		countdown = null
 	. = ..()
 
+/obj/machinery/power/supermatter_shard/examine(mob/user)
+	..()
+	if(!ishuman(user))
+		return
+
+	var/range = min(7, round(power ** 0.25))
```

Comment:
this formula should probably be a proc or something so you can use it in both places it's used